config/redis_config.py

```diff
@@ -0,0 +1,27 @@
+import os
+import redis
+from config.logger_config import logger
+
+# Redis Configuration (loaded from environment variables)
+REDIS_HOST = os.getenv("REDIS_HOST", "localhost")
+REDIS_PORT = int(os.getenv("REDIS_PORT", "6379"))
+REDIS_DB = int(os.getenv("REDIS_DB", "1"))
+
+def get_redis_connection():
```

Comment:
this should be a service, create a new folder services and inside that create redis_service.py to create redis related service class